packages/amplify-e2e-core/src/utils/nexpect.ts

```diff
@@ -749,6 +756,7 @@ export function nspawn(command: string | string[], params: string[] = [], option
   // to be able to disable CI detection we do need to pass in a childEnv
   if (options.env || pushEnv || options.disableCIDetection === true) {
     childEnv = {
+      ...childEnv,
       ...process.env,
```

Comment:
since `process.env` is already included here, what's the benefit of also having this additional logic?